pkg/monitoring/vmistats/collector.go

```diff
@@ -60,31 +65,60 @@ var (
 		},
 		nil,
 	)
+
+	instancetypeVendorLabel        = "instancetype.kubevirt.io/vendor"
+	whitelistedInstanceTypeVendors = map[string]bool{
+		"kubevirt.io": true,
+		"redhat.com":  true,
```

Comment:
Should these be constants?

Reply:
Could you also remind me what the justification was for only capturing instance types and preferences provided by certain vendors?

Reply:
since this metric is collected by telemetry systems, we want to limit the domain of possible labels to reduce the impact on Prometheus instances

Reply:
Okay, can you add that context either here or below when we are checking the whitelist?

Reply:
added